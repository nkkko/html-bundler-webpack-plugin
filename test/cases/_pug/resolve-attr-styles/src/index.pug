html
  head
    title Test
    link(href="./style.scss" rel="stylesheet")
  body
    h1 Hello World!
    img(src="@images/apple.png")
