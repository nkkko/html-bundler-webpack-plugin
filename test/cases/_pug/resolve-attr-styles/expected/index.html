<!DOCTYPE html><html><head><title>Test</title><link href="style.css" rel="stylesheet"></head><body><h1>Hello World!</h1><img src="apple.02a7c382.png"></body></html>